src/VisualStudio/LiveShare/Impl/DiagnosticHandler.Exports.cs

```diff
@@ -1,68 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Immutable;
-using System.ComponentModel.Composition;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.VisualStudio.LiveShare.LanguageServices;
-using CustomMethods = Microsoft.VisualStudio.LiveShare.LanguageServices.Protocol.CustomMethods;
-
-namespace Microsoft.VisualStudio.LanguageServices.LiveShare
-{
-    [Export(LiveShareConstants.RoslynContractName, typeof(ILspNotificationProvider))]
-    [ExportLspRequestHandler(LiveShareConstants.RoslynContractName, CustomMethods.GetDocumentDiagnosticsName)]
-    [Obsolete("Used for backwards compatibility with old liveshare clients.")]
```

Comment:
I'm all for killing the backcompat, but is this something we need to document or raise so if this breaks we're not getting feedback tickets?